_dev/src/utils/LocalStorage.ts

```diff
@@ -0,0 +1,12 @@
+export default function getDataFromLocalStorage(key: string) {
+  try {
+    const data = localStorage.getItem(key);
+    if (data !== 'undefined' && data !== null) {
+      return JSON.parse(data);
+    }
+    return null;
+  } catch (error: any) {
+    localStorage.removeItem(key);
+    throw error;
```

Comment:
To avoid breaking the application in case of error, can we send the error to the logger and return null instead?